templates/discussions-dashboard.php

```diff
@@ -103,6 +103,7 @@ function( $a, $b ) use ( $latest_comment_date_by_post_id ) {
 		<th>Project</th>
 		<th>Author</th>
 		<th>Submitted on</th>
+		<th><?php echo esc_html( apply_filters( 'gp_involved_table_heading', 'Validators Involved' ) ); ?></th>
```

Comment:
Add translation capabilities to this line.
```suggestion
		<th><?php echo esc_html__( apply_filters( 'gp_involved_table_heading', 'Validators Involved' ) ); ?></th>
```

Reply:
You can only make strings translable, not function outputs, since the strings are extracted from the plain text.